LEAF_Request_Portal/templates/reports/LEAF_import_data.tpl

```diff
@@ -14,10 +14,26 @@
     #category_indicators td, #new_form_indicators td {
         padding: 7px;
     }
+
+    .modalBackground {
+        width: 100%;
+        height: 100%;
+        z-index: 5;
+        position: absolute;
+        background-color: grey;
+        margin-top: 0px;
+        margin-left: 0px;
+        opacity: 0.5;
+    }
+
 </style>
 <script type="text/javascript" src="https://cdn.jsdelivr.net/gh/SheetJS/js-xlsx@1eb1ec/dist/xlsx.full.min.js"></script>
 <script type="text/javascript" src="https://cdn.jsdelivr.net/gh/SheetJS/js-xlsx@64798fd/shim.js"></script>
 <script type="text/javascript" src="js/lz-string/lz-string.min.js"></script>
+<link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">

```

Comment:
can we just grab the files here so whenever this importer script is used there isnt an additional script fetch.

Reply:
Please add these files under libs folder

Reply:
Done